.github/workflows/pr.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
         node-version: [16.x]
-        prisma-client-version: ['3.5']
+        prisma-client-version: ['3.10', '3.11', '3.12', '3.13']
```

Comment:
In case it helps we've implemented e2e tests in `@prisma/templates` that allow parallel testing on one database: https://github.com/prisma/templates-node/blob/main/tests/e2e/__testers__.ts

One of the key bits is the clean up operation which is handy for supporting local TDD (doesn't matter much for one-off runs in CI): https://github.com/prisma/templates-node/blob/main/tests/e2e/__testers__.ts#L38